README.md

```diff
@@ -0,0 +1,95 @@
+## What is `copr-test`
+
+The `copr-test` is the integration test for `Coprocessor` of the TiKV. The `Coprocessor` is used to execute TiDB push-down
+executor to improve the database performance. It's important to keep execution result consistent between TiDB and `Coprocessor`.
+This is the purpose of `copr-test`.
+
+The basic principle of `copr-test` is to execute test cases in TiDB standalone (with `mocktikv`) and TiDB cluster (with `Coprocessor`)
+and compare the execution result. We think the TiDB and Coprocessor is consistent if the results in both sides are same, vice versa.
+We use `github.com/pingcap/parser` to parse every file, and iterator the []ast.StmtNode to run every SQL.
+So if there is any error in the test SQL file, the error msg is hard to read. Creating the database and tables in you local database
+and use IDE(such as CLion) to write SQL file as test case is recommended.
+
+The `Coprocessor` contains two execution frameworks, `Batch` (vectorized execution engine) and `Non-Batch` in current
+stage (remove the `Non-Batch` version is in plan). So all test cases should be passed in the following execution environment.
+
+1. TiDB with mocktikv
```

Comment:
This section only applies to the push down test. Prefer to put it in the test directory instead of root directory.

Reply:
I will move it to the test directory if we add another test in this repo someday.